bedrock/mozorg/templates/mozorg/home/includes/vpn-beautiful.html

```diff
@@ -0,0 +1,50 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+
+{% call split(
+  image=resp_img(
+        url="img/home/2023/vpn-beautiful-700.jpg",
+        srcset={
+            "img/home/2023/vpn-beautiful-300.jpg": "300w",
+            "img/home/2023/vpn-beautiful-500.jpg": "500w",
+            "img/home/2023/vpn-beautiful-700.jpg": "700w",
+            "img/home/2023/vpn-beautiful-900.jpg": "900w",
+            "img/home/2023/vpn-beautiful-1100.jpg": "1100w",
+            "img/home/2023/vpn-beautiful-1300.jpg": "1300w",
+            "img/home/2023/vpn-beautiful-1500.jpg": "1500w",
+        },
+        sizes={
+            "(min-width: 768px)": "calc(50vw - 128px)",
+            "default": "calc(100vw - 48px)"
+        },
+        optional_attributes={
+          "alt": "",
```

Comment:
Nit: no need to include an empty `alt` attribute. One will always be added if a value isn't specified.